source/commands/nick.cpp

```diff
@@ -1,9 +1,38 @@
 #include "commands.hpp"
 
+bool checkNick(std::string nick)
+{
+	if (!((nick[0] >= 'a' && nick[0] <= 'z') || (nick[0] >= 'A' && nick[0] <= 'Z')))
+		return (false);
+	return (true);
+}
+
 void cmd::nick(const ft::Message& msg, ft::Client& client, ft::IRC& irc)
 {
 	(void)irc;
+	if (!client._raspberry)
+	{
+		//client.sendmsg(ft::Message(":127.0.0.1 462 :Unauthorized command (already registered)")); //ERR_ALREADYREGISTRED
```

Comment:
I'd remove the commented out line here, since actually sending that error might be misleading.

Reply:
```suggestion
//ERR_ALREADYREGISTRED
```